test_bond/test/exercise_bond.py

```diff
@@ -89,13 +89,13 @@ def test_normal(self):
         self.assertTrue(bond.wait_until_broken(rospy.Duration(5.0)))
 
     # Remote never connects
-    def test_no_connect(self):
-        id = gen_id()
-        # Don't start the other side of the bond
-        self.bond = bond = bondpy.Bond(TOPIC, id)
-        bond.start()
-        self.assertFalse(bond.wait_until_formed(rospy.Duration(1.0)))
-        self.assertTrue(bond.wait_until_broken(rospy.Duration(20.0)))
+    # def test_no_connect(self):
+    #     id = gen_id()
+    #     # Don't start the other side of the bond
+    #     self.bond = bond = bondpy.Bond(TOPIC, id)
+    #     bond.start()
+    #     self.assertFalse(bond.wait_until_formed(rospy.Duration(1.0)))
+    #     self.assertTrue(bond.wait_until_broken(rospy.Duration(20.0)))
```

Comment:
I'm assuming this test was dropped due to the nuisance of waiting for it to time out each time? Please re-enable it (you could also make it faster by monkey-patching the timeout).

Reply:
No idea how this slipped through to the PR... thanks for pointing it out.